skrub/_dataframe/_common.py

```diff
@@ -0,0 +1,22 @@
+import enum
+
+
+class Selector(enum.Enum):
```

Comment:
strings are already used for column names, so using a different type helps distinguish the 2. we could also have more complex selectors that support set operations as in polars but I thought it would be better to start with something simpler. we could also have 2 functions `select` and `select_dtypes` which both accept strings, although we might want support for selection by dtype in `SelectCols` transformers and we probably don't want to have a `SelectColsByDtype` separate transformer. how do you suggest we handle selection by dtypes?